skyrim-platform/src/platform_lib/Validators.cpp

```diff
@@ -1,12 +1,13 @@
 #include "Validators.h"
+#include <exception>
 
 bool ValidateFilename(std::string_view filename, bool allowDots)
 {
   for (char c : filename) {
     if (!(('0' <= c && c <= '9') || ('A' <= c && c <= 'Z') ||
           ('a' <= c && c <= 'z') || (c == '.' && allowDots) || c == '-' ||
           c == '_')) {
-      return false;
+      throw std::runtime_error("Error");
```

Comment:
That would make sense if this function would be written the other way. But actually a caller checks the return value and throws a meaningful (as much as it could be) exception if validation is not passed here